pvacseq/lib/main.py

```diff
@@ -229,6 +227,8 @@ def main(args_input = sys.argv[1:]):
             class_i_alleles.append(allele)
         if allele in MHCII.all_valid_allele_names():
             class_ii_alleles.append(allele)
+        if allele not in MHCI.all_valid_allele_names() and allele not in MHCII.all_valid_allele_names():
```

Comment:
Does `all_valid_allele_names` take much work to run?  Since we've already checked both MHCI and II right before this, it's possible to avoid checking them again.